Tigger/Tools/Imaging.py

```diff
@@ -261,9 +260,21 @@ def __init__(self, sproj, tproj, sl, sm, tl, tm):
         tl,tm is a (sorted, ascending) list of l,m coordinates in the target image
         """
         # convert tl,tm to to source coordinates
+        
         # find the overlap region first, to keeps the number of coordinate conversions to a minimum
-        overlap = astLib.astWCS.findWCSOverlap(sproj.wcs, tproj.wcs)
-        tx2, tx1, ty1, ty2 = overlap['wcs2Pix']
+        swcs = sproj.wcs
+        twcs = tproj.wcs
+        while swcs.pixel_n_dim > 2:
```

Comment:
This low-key assumes that the spatial coordinates will be AXIS1,AXIS2. Perhaps better to slice the WCS appropriately - https://docs.astropy.org/en/stable/api/astropy.wcs.WCS.html#astropy.wcs.WCS.slice

Reply:
This assumption is baked in throughout the code I'm afraid, not just here. I don't have the heart to go through and fix all of it. Do images with RA/DEC being different than 1/2 even exist in the wild? No radio image I've seen...